analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -231,9 +231,7 @@ class GoMod(
  * A class to represent a graph with dependencies. This representation is basically an adjacency list implemented by a
  * map whose keys are package identifiers and whose values are the identifiers of packages these packages depend on.
  */
-private class Graph(private val nodeMap: MutableMap<Identifier, Set<Identifier>>) {
-    constructor() : this(mutableMapOf())
-
+private class Graph(private val nodeMap: MutableMap<Identifier, Set<Identifier>> = mutableMapOf()) {
```

Comment:
Maybe forgot to squash this into the preceeding change?

Reply:
Oh yes. Thanks for spotting this.